Assets/MRTK/Providers/Oculus/XRSDK/Profiles/DefaultOculusXRSDKDeviceManagerProfile.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   isCustomProfile: 0
   ovrCameraRigPrefab: {fileID: 2343937678421323989, guid: 69a746aa83d0d0e45b4e2d33eab0fff4,
     type: 3}
-  renderAvatarHandsInsteadOfControllers: 1
+  renderAvatarHandsWithControllers: 0
```

Comment:
To clarify, this is changing our default to now _only_ show the controller model, not the rigged hands as well?

Reply:
Yeah, the rigged hands will no longer show up with the controller by default, since many users seemed to be confused about what exactly the avatar hands were, and it better aligns with our experience on other VR devices.